apis/wikipedia.py

```diff
@@ -5,6 +5,15 @@
 from concurrent.futures import ThreadPoolExecutor
 from constants import wikipedia_api_url
 import os
+from googlesearch import search
+
+
+def search_google(search_param: str) -> str:
+    response = search(search_param, stop=6)
+    
+    for url in response:

```

Comment:
what if there is multiple wikipedia links for a search param? Wouldn't we want to return all of them. In this case we'd only be returning the first one